contracts/OpenSTUtility.sol

```diff
@@ -601,4 +601,28 @@ contract OpenSTUtility is Hasher, OpsManaged {
     		address(registeredToken.token),
     		registeredToken.registrar);
     }
+
+	/*
+	 *  Administrative functions
+	 */
+	function initiateProtocolTransfer(
+		// including ProtocolVersioned interface explicitly adds to the size
+		// of OpenSTValue, so we make it explicitly SimpleStake which is already included.
+		UtilityTokenAbstract _token,
+		address _proposedProtocol)
+		public
+		onlyAdmin
+		returns (bool)
+	{
+		_token.initiateProtocolTransfer(_proposedProtocol);
```

Comment:
Need to explicitly return, otherwise will always return false. This comment applies to the identically-named function in `OpenSTValue.sol` as well.